docs/manage/CLI.md

```diff
@@ -99,23 +99,63 @@ To later update the Viam CLI tool, you can use the steps above to reinstall the
-Do this by issuing the command:
+Once you have [installed the Viam CLI](#install), you must authenticate your CLI session with Viam in order to run CLI commands.
 
-```sh {class="command-line" data-prompt="$"}
-viam login
-```
+You can authenticate your CLI session using either a personal access token or an organization API key.
+To use an organization API key to authenticate, you must first [create an organization API key](#create-an-organization-api-key).
+
+* To authenticate your CLI session using a personal access token:
+
+   ```sh {class="command-line" data-prompt="$"}
+   viam login
+   ```
+
+   This will open a new browser window with a prompt to start the authentication process.
+   If a browser window does not open, the CLI will present a URL for you to manually open in your browser.
+   Follow the instructions to complete the authentication process.
 
-This will open a new browser window with a prompt to start the authentication process.
-If a browser window does not open, the CLI will present a URL for you to manually open in your browser.
-Follow the instructions to complete the authentication process.
+* To authenticate your CLI session using an organization API key:
+
+   ```sh {class="command-line" data-prompt="$"}
+   viam login api-key --key-id <api-key-uuid> --key <api-key-secret-value>
+   ```
+
+   If you haven't already, [create an organization API key](#create-an-organization-api-key) to use this authentication method.
 
-{{% alert title="Info" color="info" %}}
 An authenticated session is valid for 24 hours, unless you explicitly [log out](#logout).
 
 After the session expires or you log out, you must re-authenticate to use the CLI again.
+
+### Create an organization API key
+
+To use an organization API key to authenticate your CLI session, you must first create one:
+
+1. First, [authenticate](#authenticate) your CLI session.
+   If your organization does not already have an organization API key created, authenticate using a personal access token.
+
+1. Then, run the following command to create a new organization API key:
+
+   ```sh {class="command-line" data-prompt="$"}
+   viam organization api-key create --org-id <org-id> --name <key-name>
+   ```
+
+   Where:
+
+   * `org-id` is your organization ID. You can find your organization ID by running `viam organizations list` or by visiting your organization's **Settings** page in [the Viam App](https://app.viam.com/).
+   * `key-name` is an optional name for your API key. If omitted, a name will be auto-generated based on your login info and the current time.
+
+The command will return a `key id` and a `key value`; you will need both to authenticate using `viam login api-key`
+
+{{% alert title="Important" color="note" %}}
+Secure these key values safely.
+Authenticating using an organization API key gives full write access to your organization, including to all robots within it.
```

Comment:
write access to what?

Reply:
Your robots recently-played music tracks, so it can judge it on its taste! j/k absolutely correct, will update.